client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1729,7 +1729,8 @@
   "messages.patients-found_one": "1 patient found matching the supplied details",
   "messages.patients-found_other": "{{count}} patients found matching the supplied details",
   "messages.period-not-available": "No available periods",
-  "messages.placeholder-allocated": "Not enough stock is available to allocate the requested quantity. A placeholder has been added for  {{placeholderQuantity}} units.",
+  "messages.placeholder-allocated-doses": "Not enough stock is available to allocate the requested quantity. A placeholder has been added for {{placeholderQuantity}} doses.",
+  "messages.placeholder-allocated-units": "Not enough stock is available to allocate the requested quantity. A placeholder has been added for {{placeholderQuantity}} units.",
```

Comment:
Sorry, side quest #1, was testing, noticed that when issuing in doses, warning still said it was placeholder in units.